modules/cpmso-feat-replace.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control_plane_machine_management/cpmso-using.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-feat-replace_{context}"]
+= Replacing a control plane machine
+
+You can use the `OnDelete` update strategy to replace a control plane machine manually.
+
+For clusters that are configured to use the `OnDelete` update strategy, the Operator creates a replacement control plane machine when you delete an existing machine. When the replacement control plane machine is ready, the etcd Operator allows the existing machine to be deleted. The replacement machine then joins the control plane.
+
+If multiple control plane machines are deleted, the Operator creates all of the required replacement machines simultaneously.
+
+[NOTE]
+====
+You can use the `OnDelete` update strategy to test changes to your control plane configuration. Manually replacing machines allows you to test changes to your configuration on a single machine before applying the changes more broadly.
+====
```

Comment:
Following on from my comment above, as it is correctly pointed out, this keeps being true only for the `OnDelete` strategy 👍